cartridges/app_custom_account_address/cartridge/models/account.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
Do we need this extension, isn't it ok to have just the address model extended ?

Reply:
Here in account. js i call new AddressModel with addressBook.addresses[i].raw to get the new custom attribute from address.row